src/documentation/wiki/prototyping-deck.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: Prototyping deck
```

Comment:
Perhaps change the title to 'Prototyping deck technical details'

Reply:
Yes!